CounterApp/style.css

```
@@ -0,0 +1,10 @@
+html{
+    height: 100%;
+    width: 100%;
+    background-color: brown;
+}
```

isso não devia ser no `body`?

```suggestion
body{
    height: 100%;
    width: 100%;
    background-color: brown;
}
```